fortuna/src/App.js

```diff
@@ -0,0 +1,17 @@
+//@flow strict
+
+import * as React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Login from './login/Login.js';
+
+// This builds the page.
+class App extends React.Component<{||}> {
+  
```

Comment:
This needs to store what we are rendering somewhere in its state, and then pass to children a callback method for state change at least if we aren't using flux stores.